index.js

```diff
@@ -63,13 +69,30 @@ async function getLocations(lat, lng, cursor) {
 
 const getAllPointsToCheck = async () => {
   const res = await fetch("https://maps.bookmyvaccine.covid19.health.nz/booking_site_availability.json")
+  if (res.status !== 200) {
+    Sentry.captureException(new Error("Scraper failed"), (scope) => {
+      scope.addBreadcrumb({
+        data: res.status,
+        message: res.text()
+      })
+      scope.addBreadcrumb({
+        data: res.text(),
+        message: 'Response string'
+      })
+    });
+    throw new Error("Stop scraper")
```

Comment:
Throw error from here then let it propagate all the way to main function then exit to stop committing bad data